libraries/tools/binary-compatibility-validator/reference-public-api/kotlin-stdlib-runtime-merged.txt

```diff
@@ -1,6 +1,14 @@
 public abstract interface annotation class kotlin/BuilderInference : java/lang/annotation/Annotation {
 }
 
+public final class kotlin/CharCodeJVMKt {
+	public static final fun Char-xj2QHRw (S)C
+}
+
+public final class kotlin/CharCodeKt {
+	public static final fun Char (I)C
+}
```

Comment:
Consider making these functions inline-only or merging them into the existing multi-file facades.

Reply:
made inline-only